notebooks/xvcfmerge-array-input.py

```diff
@@ -0,0 +1,126 @@
+# publish to: "terra-notebook-utils-tests" "test"
+import os
+import callysto
+
+with callysto.Cell("markdown"):
+    """
+    # xvcfmerge with `Array[String]` input
+
+    This notebook demonstrates the creation of Terra data tables as input to xvcfmerge workflows, referencing input VCFs with Array[String].
+
+    This input style is in development. xvcfmerge workflows should use version "xbrianh-input-format"
+
+    # Cohort VCF Merge
+
+    The TopMED data indexed by BioData Catalyst has been jointly called for each Freeze. TOPMed researchers not part of the TOPMed Constorium receive jointly called VCFs subsetted by project and consent code. If users have access to multiple projects and consent codes, they may wish to re-combine these VCFs. This notebook demonstrates merge tooling for jointly called VCFs.
+
+    ## Aquire the Cohort VCFs to be merged
+
+    The VCFs should be stored in directories in your workspace bucket. During this tutorial, we will use the placeholder structure
+    ```
+    gs://my-bucket/vcfsa/chr1.vcf.gz
+    gs://my-bucket/vcfsa/chr2.vcf.gz
+    ...
+    gs://my-bucket/vcfsb/chr1.vcf.gz
+    gs://my-bucket/vcfsb/chr2.vcf.gz
+    ...
+    ```
+
+    Note: Freeze5b cohort VCFs in Gen3 are currently stored as tar arhcives exactly as they are represented in dbGAP. Before running this notebook, two or more archives must be extracted using the utilities found in the [unarchive-tar-files-to-workspace notebook](terra.biodatacatalyst.nhlbi.nih.gov/#workspaces/biodata-catalyst/BioData%20Catalyst%20Collection/notebooks/launch/unarchive-tar-files-to-workspace.ipynb).
```

Comment:
Note: Cohort multi-sample VCFs are currently available in multiple formats in Gen3 with the goal of eventually hosting files in actionable formats (i.e. not tar compressed). Currently, Freeze 8 multi-sample VCFs are available as 23 files that are blocked zipped (*.vcf.gz). Some Freeze5b cohort VCFs in Gen3 may be stored as a single tar bundle. If you are interested in Freeze 5b files that are tar archived, you will need to extract the contents of the tar to your workspace using the utilities found in the [unarchive-tar-files-to-workspace notebook](https://app.terra.bio/terra.biodatacatalyst.nhlbi.nih.gov/#workspaces/biodata-catalyst/BioData%20Catalyst%20Collection/notebooks/launch/unarchive-tar-files-to-workspace.ipynb).

Reply:
fixed